graphdatascience/tests/integration/test_single_mode_algos.py

```diff
@@ -30,9 +32,19 @@ def run_around_tests(runner: Neo4jQueryRunner) -> Generator[None, None, None]:
     runner.run_query(f"CALL gds.graph.drop('{GRAPH_NAME}')")
 
 
+@pytest.mark.compatible_with(min_inclusive=ServerVersion(2, 5, 0))
 def test_triangles(gds: GraphDataScience) -> None:
     G, _ = gds.graph.project(GRAPH_NAME, "*", {"REL": {"orientation": "UNDIRECTED"}})
 
+    result = gds.triangles(G, maxDegree=2)
+
+    assert len(result) == 1
+
+
+@pytest.mark.compatible_with(max_exlusive=ServerVersion(2, 5, 0))
+def test_alpha_triangles(gds: GraphDataScience) -> None:
+    G, _ = gds.graph.project(GRAPH_NAME, "*", {"REL": {"orientation": "UNDIRECTED"}})
+
     result = gds.alpha.triangles(G, maxDegree=2)
```

Comment:
This one should also work for 2.5.0. we could check the deprecation is raised for 2.5.0 and later (with pytest.raise(...))

Reply:
still an option ;)